orc/src/main/java/org/apache/iceberg/orc/OrcFileAppender.java

```diff
@@ -153,4 +183,14 @@ private static Writer newOrcWriter(OutputFile file,
                                                          createWriterFunc) {
     return (OrcRowWriter<D>) createWriterFunc.apply(schema, orcSchema);
   }
+
+  private TreeWriter getTreeWriter() {
```

Comment:
Iceberg does not use `get` in method names because it is either not helpful or should be replaced with a more specific verb.